### PR TITLE
Add News API connector to slalpha5 branch and update documentations

### PR DESCRIPTION
**Pull request by kapilraajP**
## Automation tests
 - Unit tests 
   > Done
 - Integration tests
   > Done

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

**Review by aneeshafedo — CHANGES_REQUESTED** [review]
Module.md is missing. Please add Module.md following the guidelines given here https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?hl=en&forcehl=1

File: openapi/newsapi/Package.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Connects to News API from Ballerina
+
+#### Compatibility
+|                               | Version               |
+|-------------------------------|-----------------------|
+| Ballerina Language Version    | **Swan Lake Alpha 5** |
+|       News API Version        |         **V2**        |
+
+### Package Overview
+The <newsapi> is a [Ballerina](https://ballerina.io/) connector for News API.
```
Comment (indikasampath2000) [inline openapi/newsapi/Package.md:10]: why do you have wording like this `<newsapi>`? can't we simply say `newsapi.org`?

Comment (kapilraajP) [reply openapi/newsapi/Package.md:10]: it is the package name. can i place it as `newsapi` only instead of `newsapi.org`

**Review by indikasampath2000 — APPROVED** [review]

File: openapi/newsapi/Package.md (hunk 1 above)
Comment (indikasampath2000) [reply openapi/newsapi/Package.md]: `newsapi` is fine.

**Pull request merged**